src/ShapeDef.hs

```diff
@@ -284,6 +284,17 @@ arrowType = ("Arrow", M.fromList
         ("name", (StrT, constValue $ StrV "defaultArrow"))
     ])
 
+braceType = ("Brace", M.fromList
+    [
+        ("startX", (FloatT, x_sampler)),
```

Comment:
need to make Brace draggable as well (same problem as with Arrow)

Reply:
@hypotext , in order that a shape will be draggable, it should have `x` and `y` properties with these exact property names. So currently, many other shapes which are not mine are not draggable as well. This is a hack of @wodeni  which should be resolved in the future.